features/apimgt/org.wso2.carbon.apimgt.core.feature/src/main/resources/sql/mssql.sql

```diff
@@ -2330,6 +2330,15 @@ CREATE TABLE AM_GW_VHOST (
   PRIMARY KEY (GATEWAY_ENV_ID, HOST)
 );
 
+-- Environment to Data Plane mapping Table --
+IF NOT EXISTS (SELECT * FROM SYS.OBJECTS WHERE OBJECT_ID = OBJECT_ID(N'[DBO].[CHOREO_GW_ENV_DATA_PLANE_MAPPING]') AND TYPE IN (N'U'))
+CREATE TABLE CHOREO_GW_ENV_DATA_PLANE_MAPPING (
+  GATEWAY_ENV_UUID VARCHAR(45) NOT NULL,
```

Comment:
This is always not a UUID I suppose?

Reply:
`AM_GATEWAY_ENVIRONMENT.UUID` field is the corresponding filed to this.

Reply:
Do we get the shared environments (defined in the deployment.toml) also there? Are they added to the databases during startup?

Reply:
Environments defined in deployment.toml(read only envs) are not populated in the `AM_GATEWAY_ENVIRONMENT` table. They are not added to the table even in a re-start as well. When we call the getAllEnvironments, these read only envs are appended to the env list.